go.mod

```diff
@@ -76,6 +76,7 @@ require (
 	google.golang.org/grpc v1.62.1
 	google.golang.org/protobuf v1.33.0
 	gopkg.in/natefinch/lumberjack.v2 v2.2.1
+	gopkg.in/yaml.v2 v2.4.0
```

Comment:
I think we shouldn't be using this anymore

Reply:
I used gopkg.in/yaml.v2 to preserve the sequence of fields. I couldn't find a way to do this with sigs.k8s.io. 
Additionally, sigs.k8s.io itself uses gopkg.in/yaml.v2 internally. I believe that here you can use this library directly.

Reply:
Okay, I replaced the explicit use of gopkg.in/yaml.v2  with sigs.k8s.io/yaml/goyaml.v2